tsconfig.json

```diff
@@ -27,9 +27,15 @@
         // Output
         "outDir": "dist/",
         "rootDir": "src/",
-        "newLine": "LF"
+        "newLine": "LF",
+
+        "jsx": "react",
+        "esModuleInterop": true
```

Comment:
We can't have this on since TypeDoc is a library, by turning it on we force all our consumers to also turn it on.